### ISSUE TITLE
Injection doesn't work with --minify flag for providers

### ISSUE THREAD

**Issue by SekibOmazic** [SekibOmazic/angular-es6-jspm#2]
can't inject $http into GitHub provider.

Angular throws following exception:

Unknown provider: t

**Comment by SekibOmazic** [comment]
When built with --minify flag:

``` Javascript
t.register("lib/github/GitHubProvider",[],function(t){
    "use strict";
    var e,n;

    return {
        setters:[],
        execute:function() {
            e = function() {
                var t=function() {
                    var t=this;

                    this.$get=function(e){
                        return new n(t.baseURL,e)
                    },

                    this.$get.$inject=["$http"]
                };

                return $traceurRuntime.createClass(t, { setUrl:function(t){this.baseURL=t} }, {})
            }(),

            n = function() { 
                var t=function(t,e) {
                    this.baseURL=t,
                    this.$http=e
                };

                return $traceurRuntime.createClass(
                t, 
                {
                    contributors:function(t,e) {
                        return console.log("PROVIDER"),this.$http.get(this.baseURL+"/repos/"+t+"/"+e+"/contributors").then(function(t){return t.data})
                    }
                },

                {}
                )
            }(),

            t("GitHubProvider",e)
        }
    }
})
```

When built without --minify flag:

``` Javascript
System.register("lib/github/GitHubProvider", [], function($__export) {
  "use strict";
  var __moduleName = "lib/github/GitHubProvider";
  var GitHubProvider,
      GitHubCont;
  return {
    setters: [],
    execute: function() {
      GitHubProvider = (function() {
        var GitHubProvider = function GitHubProvider() {
          var $__0 = this;
          this.$get = (function(http) {
            return new GitHubCont($__0.baseURL, http);
          });
          this.$get.$inject = ['$http'];
        };
        return ($traceurRuntime.createClass)(GitHubProvider, {setUrl: function(url) {
            this.baseURL = url;
          }}, {});
      }());
      GitHubCont = (function() {
        var GitHubCont = function GitHubCont(url, $http) {
          this.baseURL = url;
          this.$http = $http;
        };
        return ($traceurRuntime.createClass)(GitHubCont, {contributors: function(owner, repo) {
            console.log('PROVIDER');
            return this.$http.get((this.baseURL + "/repos/" + owner + "/" + repo + "/contributors")).then((function(data) {
              return data.data;
            }));
          }}, {});
      }());
      $__export("GitHubProvider", GitHubProvider);
    }
  };
});
```

**Comment by SekibOmazic** [comment]
Fixed with jspm v0.11.2
